web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -490,6 +490,27 @@ function joinup_core_entity_create(EntityInterface $entity) {
   }
 }
 
+/**
+ * Implements hook_entity_insert().
+ */
+function joinup_core_entity_insert(EntityInterface $entity) {
+  // List the related entity bundles.
+  $related_bundles = [
+    'collection',
+    'solution',
+  ];
+
+  // Check to the current bundle is in our list.
+  if (in_array($entity->bundle(), $related_bundles)) {
+    /** @var \Drupal\Core\Messenger\MessengerInterface $messenger */
+    $messenger = \Drupal::service('messenger');
+
+    // Add a status message when it was saved.
+    $messenger->addStatus(t('Thank you for proposing a @entity_type. 
+    Your request is currently pending approval by the site administrator.', ['@entity_type' => $entity->bundle()]));
```

Comment:
As our Drupal core benefits for this patch https://www.drupal.org/project/drupal/issues/2765065, we should use the lower-cased singular form of the bundle label. `$entity->bundle()` is more a "machine name", we have now a dedicated label on the bundle entity: In this case we should use `$entity->get('rid')->entity->getSingularLabel()`, which is the "human readable" singular label to be used on UI.

Reply:
Okay! I added these kind of things into the submit callback (_joinup_core_message_handler).
Please check!